sys/include/xtimer/tick_conversion.h

```diff
@@ -12,22 +13,97 @@
  * @{
  * @file
  * @brief   xtimer tick <-> seconds conversions for different values of XTIMER_HZ
+ *
  * @author  Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ * @author  Josua Arndt <jarndt@ias.rwth-aachen.de>
  */
-
 #ifndef XTIMER_TICK_CONVERSION_H
 #define XTIMER_TICK_CONVERSION_H
 
 #ifndef XTIMER_H
 #error "Do not include this file directly! Use xtimer.h instead"
 #endif
 
+#include <stdio.h>
+#include <string.h>
+#include "fmt.h"
 #include "div.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/*
+ * As tick_conversion.h is included in multiple files ENABLE_DEBUG can not be
+ * used as it gets redefined "[-Werror]"
+ * Activate the debug print and enable module fmt in the project makefile as follow
+ *
+ * USEMODULE += fmt
+ * CFLAGS += -DENABLE_DEBUG_PRINT_TICK_CONVERSION=1
+ * */
+#ifndef ENABLE_DEBUG_PRINT_TICK_CONVERSION
+#define ENABLE_DEBUG_PRINT_TICK_CONVERSION   (0)
+#endif
+
+#if ENABLE_DEBUG_PRINT_TICK_CONVERSION
+#ifndef MODULE_FMT
+#error "Module fmt is necessary to ENABLE_DEBUG_TICKS set 'USEMODULE += fmt' \
```

Comment:
Don't wrap this line, github syntax highlighting gets confused it seems

Reply:
Also, there's no need to quote the argument to `#error `, everything on the line is printed verbatim to stderr by the preprocessor when it encounters the error directive

Reply:
Qouting is needed to wrap, if there is the no wrap, no need to qoute.

But i just try to understand this, we go ahead with code convention violations as github has a buggy code highlighting?

Reply:
Or just shorten the message, add more information in a code comment near this line. The error message will have the file name and line number so it's easy to find

Reply:
https://github.com/textmate/c.tmbundle/issues/56